src/browser-update-config.json

```diff
@@ -1,23 +1,21 @@
 {
   "required": {
-    "e": -2,
-    "f": -3,
-    "c": -3,
-    "o": -3,
```

Comment:
What's the default behaviour after removing these?

Reply:
Docs talk about falling back to a default (was assuming they would have a sensible one?). Happy to be more specific though (main worry was that damn Safari!).

Reply:
Will have a go on BrowserStack to see how sensible they are!